src/integrationTest/java/uk/gov/hmcts/reform/lrdapi/RetrieveBuildingLocationBySearchStringTest.java

```diff
@@ -39,7 +39,7 @@ void shouldRetrieveBuildingLocation_For_SearchString_WithStatusCode_200(String s
                 LrdBuildingLocationBySearchResponse[].class,
                 path
             );
-        assertThat(response).isNotEmpty().hasSize(3);
+        assertThat(response).isNotEmpty().hasSize(4);
```

Comment:
add assertion for EXTERNAL_SHORT_NAME value

Reply:
not needed for building locations

Reply:
not needed for builiding locations